net/microsocks/files/microsocks.config

```diff
@@ -0,0 +1,7 @@
+config microsocks microsocks
+	option bindaddr 192.168.33.2
```

Comment:
This IP address suits your testing or it is default?

Reply:
Yes. Just example IP#.

Reply:
Option commented out, will bind to all by default